staging/knative/charts/serving/templates/configmap-crd.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: knative-serving
+  name: knative-crd
+data:
+  crd.yaml: |-
+{{.Files.Get "crds/crds.yaml" | printf "%s" | indent 4}}
```

Comment:
do you need the printf?

Reply:
I did it by analogy with Istio (https://github.com/mesosphere/charts/blob/master/staging/istio/templates/configmap-crd.yaml#L8). I guess you need a `printf` to feed its output to `indent` but not sure about that.

Reply:
just checked that `printf` is not required here. updated the PR